config.rb

```diff
@@ -51,7 +49,11 @@
   "101/index.html"                             => "/tags/cd-101.html",
   "community/index.html"                       => "https://groups.google.com/forum/#!forum/go-cd",
   "chef/index.html"                            => "/index.html",
-  "enterprise/index.html"                      => "/index.html"
+  "events/index.html"                          => "/index.html",
+  "partners/index.html"                        => "/index.html",
+  "enterprise/index.html"                      => "/index.html",
+  "enterprise-campaign/index.html"             => "/index.html",
+  "thoughtworks-support-services-general-terms"=> "/enterprise/thoughtworks-support-services-general-terms/"
```

Comment:
Could you change this to `thoughtworks-support-services-general-terms/index.html`?

Without that, then, this will work:

https://preview.gocd.org/thoughtworks-support-services-general-terms

... but this won't:

https://preview.gocd.org/thoughtworks-support-services-general-terms/

Reply:
Ack, done in 2aecd1551b7fc04e7f86b08b4452a58f881253b3